examples/hitl/basic_viewer/basic_viewer.py

```diff
@@ -195,30 +204,21 @@ def sim_update(self, dt, post_sim_update_dict):
         self._update_help_text()
 
 
-def main():
-    args = create_hitl_arg_parser().parse_args()
-
-    if args.gui_controlled_agent_index is not None:
+@hydra.main(
```

Comment:
I wanted our HITL apps to use @hydra.main because it seems to be the popular, user-friendly way to use Hydra configs. Note the contrast with e.g. habitat_baselines run.py, which requires you to pass the config filename plus overrides on the command line (and then the config is composed later).